.github/workflows/deploy-to-cloud-run.yml

```
@@ -0,0 +1,172 @@
+        with:
+          node-version: '22'
+          cache: 'npm'
+          
+      - name: Install dependencies
+        run: npm ci
+        
+      - name: Generate Prisma client
+        run: npx prisma generate
+
+      - name: Create .env file
+        run: |
```

actionlint (1.7.7)

shellcheck reported issue in this script: SC2129:style:1:1: Consider using { cmd1; cmd2; } >> file instead of individual redirects

(shellcheck)